xbmc/interfaces/json-rpc/schema/version.txt

```diff
@@ -1 +1 @@
-6.30.0
+6.31.0
```

Comment:
Im pulling this and patching my source to give more testing, but since https://github.com/xbmc/xbmc/pull/8268 this causes previously reversed or applied patch warning since its the same version bump. not sure what the best thing todo would be in this case.